src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs

```diff
@@ -1263,10 +1337,209 @@ public static void DCJS_TypeWithKnownTypeAttributeAndListOfInterfaceMember()
     }
 
     [Fact]
+    public static void DCJS_Tuple()
+    {
+        DCJS_Tuple1();
+        DCJS_Tuple2();
+        DCJS_Tuple3();
+        DCJS_Tuple4();
+        DCJS_Tuple5();
+        DCJS_Tuple6();
+        DCJS_Tuple7();
+        DCJS_Tuple8();
+    }
```

Comment:
Our in house test infrastructure require a runproj for each test case, that's the overhead I'm referring to. You're right that one failure would prevent the rest from running, but in that case, chances are the rest will all fail.